static/js/src/advantage/subscribe/listeners/ui-event-listeners.js

```diff
@@ -16,16 +16,15 @@ export default function initUIControls(store) {
     });
   });
 
-  // Find every "purchase-modal" control (buy button and cancel button)
-  // and make them toggle the display of the purchase modal when clicked
+  // Add close modal function to window object so the React modal can use it
+  window.handleClosePurchaseModal = () => {
```

Comment:
Let's leave this as is for now, but please refactor as soon as you have some time.

I don't see a reason why the two below coudn't be merged into one entry point with a single redux store:

```js
{ productSelector: "./static/js/src/advantage/subscribe/product-selector.js",
uaSubscribe: "./static/js/src/advantage/subscribe/react/app.jsx", }
```

Reply:
I tried something similar but couldn't get it to work